skimage/transform/__init__.py

```diff
@@ -1,3 +1,9 @@
+"""Utilities to perform various transforms
+
+The following transforms are available:
```

Comment:
Thanks @rum1887. Looks already quite good. Although rather than this line, what do you think about a short text summary of the kind of transforms that are available and what they might be used for. That might require some looking at the functions in [skimage.transform](https://scikit-image.org/docs/dev/api/skimage.transform.html) or reading the linked gallery examples but I think it would be a very good exercise to get more familiar.

Reply:
yes, on it!

Reply:
Thanks for the review @lagru , I have categorised the transforms and listed the routines available under each category! Lemme know if it looks good now.